core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -908,10 +974,18 @@ public void completeFile(AlluxioURI path, CompleteFileOptions options)
       InvalidFileSizeException, FileAlreadyCompletedException, AccessControlException {
     Metrics.COMPLETE_FILE_OPS.inc();
     try (JournalContext journalContext = createJournalContext();
-        LockedInodePath inodePath = mInodeTree.lockFullInodePath(path, InodeTree.LockMode.WRITE)) {
-      mPermissionChecker.checkPermission(Mode.Bits.WRITE, inodePath);
+         LockedInodePath inodePath = mInodeTree.lockFullInodePath(path, InodeTree.LockMode.WRITE);
+         MasterAuditContext auditContext = createAuditContext("completeFile", path, null,
+             inodePath.getInode())) {
```

Comment:
Ideally the audit logger would be like an observer taking notes about the state of RPCs in the system, not a potential source of exceptions. If we always use "safe" (non-exception-throwing) methods for looking up inodes, we don't need to worry at all about whether the audit logger will throw the same exception that would have happened anyway.